kinesis-video-gstreamer-plugin/src/gstkvssink.cpp

```diff
@@ -1001,9 +994,9 @@ void create_kinesis_video_frame(Frame *frame, const nanoseconds &pts, const nano
 
 bool
 put_frame(shared_ptr<KinesisVideoStream> kinesis_video_stream, void *frame_data, size_t len, const nanoseconds &pts,
-          const nanoseconds &dts, FRAME_FLAGS flags, uint64_t track_id, uint32_t index) {
+          const nanoseconds &dts, FRAME_FLAGS flags, uint64_t track_id, uint32_t index, uint32_t timecode_scale_ms) {
     Frame frame;
-    create_kinesis_video_frame(&frame, pts, dts, flags, frame_data, len, track_id, index);
+    create_kinesis_video_frame(&frame, pts, dts, flags, frame_data, len, track_id, index, timecode_scale_ms);
```

Comment:
I see, here we don't pass any pointers so specifying timecode scale is OK I guess but you can specify the pre-computed value.

Reply:
removed timecode scale.